### ISSUE TITLE
При запуске epf VA в предеприятии - показывать сообщение пользователю, если не использован ключ /TestManager

### ISSUE THREAD

**Issue by lelikanet** [Pr-Mex/vanessa-automation#1402]
Сообщать пользователю, что предприятие запущено без ключа /TestManager если это действительно так. Можно ещё предложить пользователю проверить снята ли защита от опасных действий.

**Comment by nixel2007** [comment]
я понимаю, что работа с тест клиентом - это основной сценарий, но все же с VA можно работать и без тест-менеджера.
Насколько я помню, сейчас при попытке запуска тест клиента уже есть ругань на отсутствие флага. разве этого не хватает?

**Comment by Pr-Mex** [comment]
> я понимаю, что работа с тест клиентом - это основной сценарий, но все же с VA можно работать и без тест-менеджера. Насколько я помню, сейчас при попытке запуска тест клиента уже есть ругань на отсутствие флага. разве этого не хватает?

@nixel2007 
Да, ты прав. Сейчас сообщение выдается при попытке подключить клиента, что нет ключа TestManager.

**Comment by lelikanet** [comment]
Новички о тонкостях не знают. И сразу хотят запустить сценарии. И не понимают что клиент не запускается потому что сценарий написан и пытается запуститься не из менеджера.

**Comment by nixel2007** [comment]
> И не понимают что клиент не запускается потому что сценарий написан и пытается запуститься не из менеджера.

так об этом же прям в сообщении об ошибке написано...
